### DISCUSSION TITLE
Automatic Frames Between Scenes

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#348] [Ideas]
Just an idea, but what if there was a feature in the rendering section where we can tell it how many frames or seconds should be in between scenes when rendering the whole project, that way we won't have to manually make each project a second or two longer to give each animation breathing room. The frames it would add would basically be the last frame on each project held for a specific amount of time, then it would jump to the next scene.

**Comment by VeiloPot** [comment]
> Just an idea, but what if there was a feature in the rendering section where we can tell it how many frames or seconds should be in between scenes when rendering the whole project, that way we won't have to manually make each project a second or two longer to give each animation breathing room. The frames it would add would basically be the last frame on each project held for a specific amount of time, then it would jump to the next scene.

Definitely would want to see this !!!
